src/cascadia/TerminalCore/ColorFix.cpp

```diff
@@ -0,0 +1,280 @@
+#include "pch.h"
+
+#include <Windows.h>
+#include <math.h>

```

Comment:
Looks like we already have `#include <cmath.h>` in `LibraryIncludes.h` (already included via the `pch.h` above). Curious what you need `math` for.